src/Liip/RMT/Prerequisite/TestsCheck.php

```diff
@@ -25,7 +25,7 @@ class TestsCheck extends BaseAction
     public function __construct($options)
     {
         $this->options = array_merge(array(
-            'command' => 'phpunit --stop-on-failure',
+            'command' => './bin/phpunit --stop-on-failure',
```

Comment:
@ragusa87 Can we be sure that we will found a local bin script? According to https://phpunit.de/manual/6.5/en/textui.html the `phpunit` script must be in the path

Reply:
@jeanmonod  You are right, I think it's best if you keep it as it was and drop the commit as I am not sure of what I did.

Furthermore `./bin/phpunit` is not standard. It should have been something like that: `./vendor/bin/phpunit`

Reply:
Can you fix your PR then?

Reply:
done ;)

Reply:
However, testing is still required...